igvmmeasure/Cargo.toml

```
@@ -10,7 +10,7 @@ clap = { workspace = true, default-features = true, features = ["derive"] }
 sha2.workspace = true
 igvm.workspace = true
 igvm_defs.workspace = true
-p384.workspace = true
+p384 = { version = "0.13.0" }
```

```suggestion
p384 = { workspace = true, default-features = true }
```

But if you want to remove this change at all, you can just avoid to disable default features for `p384` in the main Cargo.toml and do that only in `kernel`, which I think should be the best approach also for the other dependencies, at least where we think we need to disable default features just in `kernel` usually to enable `alloc`

---

I wasn't able to avoid disabling the default features in `Cargo.toml`, as leaving them enabled yet disabling them in `kernel/Cargo.toml` threw an error. Because of this, I added your suggested change instead:

`p384 = { workspace = true, default-features = true }`